nat-lab/tests/utils/connection_tracker.py

```diff
@@ -199,5 +293,5 @@ async def run(self) -> AsyncIterator["ConnectionTracker"]:
             await asyncio.sleep(0.1)
```

Comment:
Unfortunately conntrack takes a bit of time to start, without the sleep the sync event might arrive before stdout_callback is ready to process it.. An alternative I guess would be to ping it in a loop.

Reply:
I checked what is the underlying problem. It's not that ping is not sending pings continuously, but our conntrack wrapper reacts only on [NEW] events. Because all pings sent from the same command execution have the same id (different seq number but the same id) they are treated as one connection. Because of that it's possible that we do not catch this first ping and hance we are missing NEW event. Your solution now should work but I think it can be simplified to something like this (not tested. Written in the browser ;)):
```
while not self._sync_event.is_set():
    await ping(self._connection, "127.0.0.2")
```